README.md

```diff
@@ -1,3 +1,109 @@
-# cordova
+# Apache Cordova
 
-Apache Cordova
+- Mobile apps with HTML, CSS & JS
+- Target multiple platforms with one code base
+- Free and open source
+
+## Repositories
+
+A list of all repositories on GitHub: 
+
+### Platforms
+
+- https://github.com/apache/cordova-android
+- https://github.com/apache/cordova-ios
+- https://github.com/apache/cordova-windows
+- https://github.com/apache/cordova-browser
+- https://github.com/apache/cordova-test-platform
+
+#### Deprecated Platforms
+
+- https://github.com/apache/cordova-blackberry
+- https://github.com/apache/cordova-osx
+- https://github.com/apache/cordova-firefoxos
+- https://github.com/apache/cordova-ubuntu
+- https://github.com/apache/cordova-wp8
+- https://github.com/apache/cordova-tizen
+- https://github.com/apache/cordova-qt
+- https://github.com/apache/cordova-webos
+- https://github.com/apache/cordova-amazon-fireos
+- https://github.com/apache/cordova-wp7
+- https://github.com/apache/cordova-bada
+- https://github.com/apache/cordova-electron
+- https://github.com/apache/cordova-bada-wac
+
+### Tools
+
+- https://github.com/apache/cordova-js
+- https://github.com/apache/cordova-plugman
+- https://github.com/apache/cordova-lib
+- https://github.com/apache/cordova-cli
+- https://github.com/apache/cordova-coho
```

Comment:
I feel like it would make sense to leave coho in tools, but to also add it to the docs section. Specifically linking to https://github.com/apache/cordova-coho/blob/master/docs

Reply:
Very good point.

Reply:
added